js/csv_to_html_table.js

```diff
@@ -29,6 +31,22 @@ CsvToHtmlTable = {
                 for (var headerIdx = 0; headerIdx < csvHeaderRow.length; headerIdx++) {
                     $tableHeadRow.append($("<th></th>").text(csvHeaderRow[headerIdx]));
                 }
+
+                if (column_filters) {
+                    var $tableHeadRow_filter = $("<tr></tr>");
+                    for (var headerIdx = 0; headerIdx < csvHeaderRow.length; headerIdx++) {
+                        $tableHeadRow_filter.append($("<th></th>").html(
+                            '<input type="text" id="col_f-' + headerIdx + '" placeholder="filter column" />'));
+                        $tableHeadRow_filter[0].childNodes[headerIdx].childNodes[0].onchange = function(ee) {
+                            var i = parseInt(ee.target.id.substr(6));
+                            CsvToHtmlTable_DataTable.column(i).search(this.value).draw();
```

Comment:
looks like this is a new vaiable to keep track of the `DataTable` state. Is it necessary for this code to work? It seems a bit awkward to only use it for this feature. Maybe we should keep track of the `DataTable` this way generally?